packages/ethernaut-interact/src/tasks/calldata.js

```diff
@@ -0,0 +1,71 @@
+const { ethers } = require('ethers')
+const types = require('ethernaut-common/src/validation/types')
+const output = require('ethernaut-common/src/ui/output')
+const EthernautCliError = require('ethernaut-common/src/error/error')
+const abi = require('../../../ethernaut-interact-ui/src/suggest/abi')
```

Comment:
ui extension packages are not intended to be imported in their base package.
They are intended to be completely standalone from the extension.